src/util/expr.h

```diff
@@ -37,8 +37,17 @@ class depth_iteratort;
 class const_depth_iteratort;
 class const_unique_depth_iteratort;
 
-/*! \brief Base class for all expressions
-*/
+/// Base class for all expressions.
+/// Inherits from \ref irept and has operands (stored as unnamed children of
+/// `irept`), and a type (which is a named sub with identifier `ID_type`).
```

Comment:
:pick: In `util/README.md` the style seems to be to always use links (i.e. `\ref irept`) rather than first a link and then just monospace. Not sure if that's a standard though.